src/ProductFeedStatus.php

```diff
@@ -92,7 +98,47 @@ public static function set( $state ) {
 	public static function deregister() {
 
 		foreach ( self::STATE_PROPS as $key => $default_value ) {
+			self::$state[ $key ] = $default_value;
 			delete_transient( self::PINTEREST_FOR_WOOCOMMERCE_FEEDS_DATA_PREFIX . $key );
 		}
 	}
+
+	/**
+	 * Resets a feed generation start time.
```

Comment:
Please add `@since` to the methods.
You can use `@since x.x.x` and the release process will automatically populate the version.